host-controller/src/main/java/org/jboss/as/domain/controller/operations/DomainServerLifecycleHandlers.java

```diff
@@ -72,6 +73,7 @@
             .build();
 
     private static final AttributeDefinition TIMEOUT = SimpleAttributeDefinitionBuilder.create(ModelDescriptionConstants.TIMEOUT, ModelType.INT, true)
+            .setValidator(new IntRangeValidator(-1))
```

Comment:
This is incorrect. The value of zero is legal. It's the default.

I see no reason to make an API change to disallow values < -1. If this were a new API I'd agree that limiting to -1 is better, but this API was part of WildFly 9.